client/src/views/EventsView.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="!isLoading">
+  <div class="container" v-if="!isLoading">
```

Comment:
I think just changing it from a div to a v-container should work
` <v-container v-if="!isLoading"> `